src/main/config/config.c

```diff
@@ -127,6 +127,7 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
     .hseMhz = SYSTEM_HSE_MHZ,  // Only used for F4 and G4 targets
     .configurationState = CONFIGURATION_STATE_UNCONFIGURED,
     .enableStickArming = false,
+    .statsSaveMoveLimit = 20,
```

Comment:
There is a persistent stats parameter group in [src/main/pg/stats.c](https://github.com/betaflight/betaflight/blob/master/src/main/pg/stats.c). It's probably better to move the parameter there.